test/server/listener_manager_impl_quic_only_test.cc

```diff
@@ -44,10 +55,13 @@ TEST_F(ListenerManagerImplQuicOnlyTest, QuicListenerFactoryAndSslContext) {
 reuse_port: true
 udp_listener_config:
   udp_listener_name: "quiche_quic_listener"
+udp_writer_config:
+  name: "udp_gso_batch_writer"
```

Comment:
Add another test to test default initialization.

Reply:
Added a test `UdpDefaultWriterConfig` under `ListenerManagerImplTest` to test default initialization for udp writer.